core/domain/prod_validation_jobs_one_off.py

```diff
@@ -2737,9 +2737,11 @@ class GeneralFeedbackThreadUserModelValidator(BaseModelValidator):
     @classmethod
     def _get_model_id_regex(cls, unused_item):
         # Valid id: [user_id].[thread_id]
-        thread_id_string = '%s\\.[A-Za-z0-9-_]{1,%s}\\.[A-Za-z0-9-_=]{1,}' % (
-            ('|').join(suggestion_models.TARGET_TYPE_CHOICES),
-            base_models.ID_LENGTH)
+        thread_id_string = '%s\\.[A-Za-z0-9-_]{1,%s}\\.[A-Za-z0-9-_]{1,%s}' % (
```

Comment:
I know this regex was already here, but are you able to leave a comment roughly describing what the regex does?